tests/chainer_tests/test_variable.py

```diff
@@ -507,6 +507,17 @@ def test_unchain(self):
         assert ret[1].rank == old_rank
         self.check_backward((ret[1],), (ret[2],), (ret[3],), False)
 
+    def test_unchain_split(self):
+        if not len(self.x.shape) > 0:
+            return
+        ret = get_variable(np, self.x)
+        ret.grad = np.zeros_like(ret.data)
+        y1, y2 = F.split_axis(ret, [5], axis=0)
+        y1.unchain()
+        z1, z2 = F.sum(y1), F.sum(y2)
+        z1.backward()
+        self.check_backward((ret,), (y1, y2,), (z2,), False)
```

Comment:
To test `y1.unchain()`, it's more natural to test that `y1` become an "input".  While it'll be less realistic, I recommend something like
```
w = z1 + z2
self.check_backward((ret, y1), (y2, z1, z2), (w,), False)
```

Reply:
Okay, this is better.